test/clj/game_test/cards/events.clj

```diff
@@ -393,7 +393,9 @@
       (prompt-choice :runner "HQ")
       (is (= 4 (:rec-counter (find-card "Cold Read" (get-in @state [:runner :play-area])))) "Cold Read has 4 counters")
       (run-successful state)
-      (card-ability state :runner (get-program state 0) 0)
+      (prompt-choice :runner "Yes")
+      (prompt-choice :runner "Yes")
+      ; (card-ability state :runner (get-program state 0) 0)
```

Comment:
Delete this comment if not needed now?

Reply:
I'd definitely remove it for the final PR. This was an "I'm annoyed and don't want to work on this at the moment." commit, thus commented code instead of flatly removed.